### ISSUE TITLE
CLI commands inconsistent

### ISSUE THREAD

**Issue by marklawlor** [jaws-framework/JAWS#176]
The CLI usage is inconsistent across commands.  They either have the format of

`$ jaws <action> <context>` eg. `$jaws new project`
or 
`$ jaws <context> <action>` eg. `$jaws module create`

If a standard command format can be picked, it would make the CLI user experience much nicer.

I believe the commands should be in the format of `$jaws <context> <action>` as it allows for easier future growth and better contextual -h/--help documentation.

**Comment by ac360** [comment]
:+1: Good catch @marklawlor

**Comment by ac360** [comment]
Fixed in v1.3.  Thanks @shortjared and @marklawlor
